src/components/top-bar/styles.ts

```diff
@@ -2,6 +2,22 @@ import { styled } from '@linaria/react'
 import { isBelowWideScreen, isTablet, isBelowDesktop, isWideScreen } from '../../styles/media'
 import { ElButtonGroup } from '../button-group'
 import { css } from '@linaria/core'
+import { ElNavResponsiveAppSwitcherIconWrap } from '../nav'
+
+export const ElTopBarAppSwitcher = styled.a`
+  padding-right: var(--spacing-4);
+
+  ${isWideScreen} {
+    padding-right: var(--spacing-5);
+  }
+
+  // TODO: can be removed once new app-switcher ready
+  ${ElNavResponsiveAppSwitcherIconWrap} {
+    margin-right: 0;
+    padding: var(--spacing-half);
+    box-sizing: content-box;
```

Comment:
so that the additional padding doesn't reduce the size of the component

Reply:
**question:** @nurm717123 can you post some screenshots that show the problem here?

**suggestion:** I'd prefer we don't add mess to our TopBar just to get a deprecated component looking good. Can we push these styles into `ElNavResponsiveAppSwitcherIconWrap` via an optional prop or something? That way the mess lives with the component that owns it 🤔

Reply:
Thanks for the screenshots 🙏  I think your suggestion to provide a class exported from `NavResponsiveAppSwitcher` that consumers can manually apply is fine, though I don't see how it's any harder to add a prop that just applies the extra class instead of making `className` available and expect consumers to use the new class. Either way though, we're keeping the mess inside the old component, which is great 👍

Reply:
its not that hard actually, I think its easier because that way I don't have to add unit test 😅